account_payment_order/models/account_payment_order.py

```diff
@@ -330,7 +330,7 @@ def generated2uploaded(self):
         return True
 
     @api.multi
-    def _prepare_move(self):
+    def _prepare_move(self, bank_lines=None):
```

Comment:
It's not used in this module, but it is useful when overriding `_prepare_move' so some parts of the move can be customized in extension modules.